armi/nuclearDataIO/tests/test_xsLibraries.py

```diff
@@ -245,6 +245,24 @@ def test_mergeFailsWithNonIsotxsFiles(self):
         finally:
             os.remove(dummyFileName)
 
+        dummyFileName = "ISOtopics.txt"
+        with open(dummyFileName, "w") as file:
+            file.write(
+                "This is a file that starts with the letters 'ISO' but will"
+                " break the regular expression search."
+            )
+
+        try:
+            with mockRunLogs.BufferLog() as log:
+                lib = xsLibraries.IsotxsLibrary()
+                xsLibraries.mergeXSLibrariesInWorkingDirectory(lib, "ISOTXS", "")
```

Comment:
I think the inputs into this function should be (lib, "", False) because there shouldn't be any library suffix required for this to fail. Additionally, the final kwargs is a boolean even though an empty string does resolve to False. Can you try this to see if the test still fails?

Reply:
Yeah, you're right about the inputs. I just copied them from the test above this one, but you're right that they are weird, so I changed them to use the defaults and pushed a new version of the test just now.

Reply:
Also yes, the test still passes with the updated inputs.